src/app.ts

```diff
@@ -1,13 +1,13 @@
-import { Workflow } from './workflow.interface';
+import { defineWorkflow } from './workflow.interface';
 import { WorkflowExecutor } from './workflow';
 
-const workflow: Workflow = {
+const workflow = defineWorkflow({
   nodes: [
     {
       id: 'A',
       execute: () => {
         console.log('A');
-        return { $next: 'false' };
+        return { $next: 'true' };
```

Comment:
auto-completion now works. it also works on `nodeById()`. `createEdgesByFrom()` could be improved too I think